buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1389,5 +1402,39 @@ artifactId=${project.name}
         args argsNeeded
       }
     }
+
+
+    /** ***********************************************************************************************/
+
+    // Method to create the PortableValidatesRunnerTask.
+    project.ext.createPortableValidatesRunnerTask = {
+      def config = it ? it as PortableValidatesRunnerConfig : new PortableValidatesRunnerConfig()
+      def name = config.name ? config.name : "validatesPortableRunner"
+      def testCategories = {
+        includeCategories 'org.apache.beam.sdk.testing.ValidatesRunner'
+        excludeCategories 'org.apache.beam.sdk.testing.FlattenWithHeterogeneousCoders'
+        excludeCategories 'org.apache.beam.sdk.testing.LargeKeys$Above100MB'
+        excludeCategories 'org.apache.beam.sdk.testing.UsesCommittedMetrics'
+        excludeCategories 'org.apache.beam.sdk.testing.UsesGaugeMetrics'
+        excludeCategories 'org.apache.beam.sdk.testing.UsesDistributionMetrics'
+        excludeCategories 'org.apache.beam.sdk.testing.UsesAttemptedMetrics'
+        excludeCategories 'org.apache.beam.sdk.testing.UsesTimersInParDo'
+        excludeCategories 'org.apache.beam.sdk.testing.UsesTestStream'
+      }
+      testCategories = config.testCategories ? config.testCategories : testCategories
+      project.tasks.create(name: name, type: Test) {
+        group = "Verification"
+        description = "Validates the PortableRunner with JobServer ${config.jobServerDriver}"
+        systemProperty "beamTestPipelineOptions", JsonOutput.toJson([
+          "--runner=org.apache.beam.runners.reference.testing.TestPortableRunner",
+          "--jobServerDriver=${config.jobServerDriver}",
+          config.jobServerConfig ? "--jobServerConfig=${config.jobServerConfig}" : "",
+        ])
+        classpath = project.configurations.validatesPortableRunner
```

Comment:
Please either document the requirement of the existence of a `validatesPortableRunner` configuration or make the configuration pluggable.

Reply:
I will make it pluggable and document it.